src/components/general/Schedule.vue

```diff
@@ -1,28 +1,34 @@
-<template>
-	<h2>Schedule</h2>
-	<router-link v-if="link" to="/schedule">View full schedule ></router-link>
-	<div>
-		<schedule-tile title="hi"/>
-	</div>
-</template>
-
-<script>
-import ScheduleTile from "@/components/general/ScheduleTile.vue";
-
-export default {
-	name: 'Schedule',
-	components: {
-		ScheduleTile
-	},
-	props: {
-		link: Boolean
-	},
-}
-</script>
-
-<style scoped>
-a {
-	color: var(--main-color);
-	text-decoration: underline;
-}
+<template>

```

Comment:
just a heads up but before merging, could you not commit this file! This would cause merge conflicts with my pr but @draymond63 thoughts?

Reply:
Agreed, please remove this from the commit history. You might have to revert the commit to do so.